docs/13_0_0/components/dashboard.md

```diff
@@ -26,6 +26,7 @@ Dashboard provides a portal like layout with drag&drop based reorder capabilitie
 | model | null | DashboardModel | Dashboard model instance representing the layout of the UI.
 | disabled | false | Boolean | Disables the component.
 | reordering | true | Boolean | Allow reordering of panels.
+| responsive | false | Boolean | In responsive mode, allows use of PrimeFlex CSS to control widget sizes and responsiveness.
```

Comment:
Why not true by default? As the other mode is legacy it will probably be removed at some point. And we already introduced breaking changes on this component.

Reply:
I always vote that it is not a good idea to break backwards compatibility as long as a deprecation is still pending. In my understanding responsive="false" is even not yet deprecated. Also, as I understood @melloware this is a non breaking change, right?

Reply:
Correct i wanted everyone using the current dashboards NOT to be broken.